packages/bot/src/views/components/commit-list.pug

```diff
@@ -0,0 +1,24 @@
+- let page = 0
+h3 Violated Commits
+p
+  b Total Count:
+  |  #{totalCount}
+ul.list-group
+  each commit in commits
+    li.list-group-item.rounded-0.flex-column.align-items-start
+      .d-flex.w-100.justify-content-between
+        a.mb-1(href=`/dashboard/commit/${commit.id}`) #{commit.repo_name}@#{commit.id.slice(0, 8)}
+        small #{commit.score} score
+      p.mb-1.font-italic.text-truncate "#{commit.message.split("\n")[0]}"
+      small #{commit.committed_at}
+ul.pagination.mt-2.d-flex.flex-wrap
+  li.page-item(class=currPage === 1 ? "disabled" : "")
+    a.page-link(href=`?page=${currPage - 1}` aria-label='Previous')
+      span(aria-hidden='true') &laquo;
+  while page < pageCount
+    li.page-item(class=currPage === ++page ? "active" : "")
```

Comment:
I would prefer move the increment operator to the loop condition to be clearer. Does pug support that?

Reply:
No, but I can move the increment operator to the end of the while block